nj2k/src/org/jetbrains/kotlin/nj2k/conversions/DefaultArgumentsConversion.kt

```diff
@@ -125,6 +133,52 @@ class DefaultArgumentsConversion(context: NewJ2kConverterContext) : RecursiveApp
 
     }
 
+    private fun JKExpression.addThisReceiverIfNeeded(
```

Comment:
Also, please use `applyRecursive` for creating recursive conversions as it will not unnecessary recreate tree elements